chapters/Chapter 1 — Untitled.md

```diff
@@ -18,18 +18,21 @@ Leanne turned just in time to see a crate of batteries break free of its binding
 
 Small batteries spilled from the crate as it smashed against the wall. Leanne did her best to ignore them as they pummeled her. Her legs and arms strained as she climbed into the cockpit.
 
-She reached for the _Engine Power_ switch. If she turned the engines on and managed to wrestle the controls, she could right the ship and get away. But Leanne could never get away. They were always after her, and they always would be. She could kill two today, and two more would take their place tomorrow. But if she let them live, they’d be back, and so would the two others, and so would more besides.
+She reached for the _Engine Power_ switch. If she turned the engines on and managed to wrestle the controls, she could right the ship and get away. But Leanne could never get away: they were always after her, and they always would be. There were at least two after her today, and there’d be at least two more tomorrow, and two more after that. 
```

Comment:
Is it "too much" if this sounds more desperate? 

> She reached for the _Engine Power_ switch. If she turned on the engines, if she managed to wrestle the controls, she could right the ship, she could get away. 

The last part could be two sentences: 

> She reached for the _Engine Power_ switch. If she turned on the engines, if she managed to wrestle the controls, she could right the ship. She could get away. 

Should the next part be its own paragraph?

> But Leanne could never get away. They were always after her. They always would be. There were at least two after her today, and there'd be at least two more tomorrow.

I like starting with the shorter sentences (very matter-of-fact, rather depressing), leading into the longer multi-clause sentence that then seems more desperate and helpless.